pages/templates/categories/_slug.vue

```diff
@@ -19,18 +19,75 @@ export default {
       { params: { category_slug: params.slug } }
     )
 
-    const { data: categories } = await $axios.get(
+    let { data: categories } = await $axios.get(
       'https://app.formester.com/template_categories.json'
     )
-
+    categories = {
+      industry: [
+        {
+          id: 23,
+          name: 'Order Forms',
+          is_active: true,
+          created_at: '2023-10-20T13:45:21.191Z',
+          updated_at: '2023-10-20T15:42:32.427Z',
+          slug: 'order-forms',
+          kind: 'industry',
+        },
+      ],
```

Comment:
same here as well should it remain here?

Reply:
No, it will be removed also